typescript/src/util.ts

```diff
@@ -63,6 +63,12 @@ String.prototype.capitalize = function(this: string): string {
   }
   return this[0].toUpperCase() + this.slice(1);
 };
+String.prototype.capitalizeFirst = function(this: string): string {
```

Comment:
We already have `.capitalize` that does _exactly_ this :)

Reply:
On nice, thought it's capitalizing all letters, great! I will remove this helper function, thanks!